posts/2023-07-13-installer-security-fixes.rst

```diff
@@ -0,0 +1,25 @@
+.. post:: 2023-07-13
+   :tags: security
+   :author: conda-forge
+
+.. role:: raw-html(raw)
+   :format: html
+
+Security updates to our installers
+==================================
+
+In June 2023, software engineers from `Anaconda <https://www.anaconda.com>`__ have reported a security issue in the uninstallers that are included in the Windows versions of the `miniforge and mambaforge installers <https://github.com/conda-forge/miniforge>`__, one of the main ways to bootstrap conda-forge based conda & mamba distributions.
+
+The issue could, under specific conditions, unintentionally delete files from your system during the uninstallation process. Anaconda has published more details in the related `blogpost <https://www.anaconda.com/blog/windows-installer-security-fix>`__ about the security fix for the miniconda and Anaconda Distribution Windows installers as well.
+
+Conda-forge is committed to fix the miniforge and mambaforge installers equally to reduce the possible impact on conda-forge users and has worked with Anaconda to mitigate the issue.
+
+- As such, we are strongly recommending all users of miniforge and mambaforge to **update immediately** to the latest versions of miniforge and mambaforge. Please download them from the `miniforge repository's main page <https://github.com/conda-forge/miniforge>`__ or the `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.3.0-1>`__.
+
+- For older versions, we are providing a **security patch for already installed miniforge and mamba installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.3.0-1>`__ as well, under the names ``miniforge-uninstaller-patch-win-64-2023.07-0.exe`` and ``mambaforge-uninstaller-patch-win-64-2023.07-0.exe``.
```

Comment:
I think there have been some issues with 23.3.0, because the last tag is 23.1.0 (even if the last files point to 23.3.0, the version is overridden by the tag, I think).

@isuruf @xhochy - can you clarify which Miniforge version is the one that is guaranteed to work upon a new release? 23.3.0 or 23.1.0?